.github/workflows/continous-integration.yml

```diff
@@ -187,5 +195,6 @@ jobs:
       env:
         SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
       uses: Ilshidur/action-slack@2.1.0
+      if: success()
       with:
         args: "⚡ New *Rasa SDK* version {{ github.event.release.tag_name }} has been released! Changelog: https://github.com/RasaHQ/rasa-sdk/blob/{{ github.event.release.tag_name }}/CHANGELOG.mdx"
```

Comment:
The `github.event.release.tag_name` field is only available for a `release` event. We don't use this kind of event for the CI workflow.

I'd create an additional step that publishes as output a tag name and then you can use it in the slack messeges.